nextflow.config

```diff
@@ -34,6 +34,7 @@ params {
     eggnog_dbpath               = "./eggnog/"
 
     // eukulele
+    eukulele_download_database  = null
     eukulele_db                 = null
```

Comment:
I wonder if we should have a default on `eukulele_db`, since the default will be that it gets run? I'd vote for `gtdb`.

Reply:
right. this was an old test. I will remove it. I will work today on a new `EUKulele` based on what we decided this week.